src/strategies/base.ts

```diff
@@ -333,51 +333,96 @@ export abstract class BaseStrategy implements Strategy {
     };
   }
 
-  protected extraFileUpdates(
+  // Helper to convert extra files with globs to the file paths to add
+  private async extraFilePaths(extraFile: ExtraFile): Promise<string[]> {
+    if (typeof extraFile !== 'object') {
```

Comment:
Why this happens? Does something pass string to this function?

Reply:
Yes, this config is coming from a config file which is an array of a set of different types -- one of which is a string.

https://github.com/googleapis/release-please/blob/main/schemas/config.json#L114-L178